### PR TITLE
Loading updates

### PR DESCRIPTION
**Pull request by martypowell**
Oops I did it again, another huge one sorry guys. I will do better

### CODE REVIEW

File: src/Card/CardList.js [hunk 1]
```diff
@@ -13,19 +15,32 @@ const defaultProps = {
 
 export default class CardList extends React.Component {
     render() {
-        const { cardContentComponent: CardContentComponent } = this.props;
+		const { cardContentComponent: CardContentComponent } = this.props;
```
Comment (jsmith176) [inline src/Card/CardList.js]: Spacing gets a little wonky here.

File: src/Filter/Filter.css [hunk 2]
```diff
@@ -19,8 +19,9 @@
 }
 
 .filter-list li label {
+	display: flex;
```
Comment (jsmith176) [inline src/Filter/Filter.css]: Spacing is off here too

File: src/Filter/MultipleSelectFilter.js [hunk 3]
```diff
@@ -28,15 +28,13 @@ export default class MultipleSelectFilter extends React.Component {
...
+                        {type === 'multiple' && options.map((option, index) => (
                             <li key={index}>
-                                {type === 'multiple' && (
-                                    <FilterCheckbox
-                                        onClick={onChange}
-                                        label={option.label}
-                                        value={option.value}
-                                    />
-                                )}
+								<FilterCheckbox
```
Comment (jsmith176) [inline src/Filter/MultipleSelectFilter.js]: Spacing

File: src/LoadMore/LoadMore.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	onSelect: () => {}
+};
+
+export default class LoadMore extends React.Component {
+    constructor(props) {
+        super(props);
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+	handleClick() {
```
Comment (jsmith176) [inline src/LoadMore/LoadMore.js]: Spacing

**Comment by martypowell** [comment]
@jsmith176 ready for another look

**Pull request merged**